internal/api/web/views/components/quiz_components/answers.templ

```diff
@@ -0,0 +1,56 @@
+package quiz_components
+
+import "github.com/Molnes/Nyhetsjeger/internal/data/questions"
+import "fmt"
+
+type Optional struct {
+	correctAnswered []string
+}
+
+type Option func(*Optional) Optional
+
+func CorrectAndAnswered(correct string, answered string) Option {
+	return func(o *Optional) Optional {
+		o.correctAnswered = append(o.correctAnswered, correct, answered)
+		return *o
+	}
+}
```

Comment:
This should either be refactored to do stuff another way, or have comments explaining what it does.